presto-docs/src/main/sphinx/sql/insert.rst

```diff
@@ -35,3 +36,14 @@ Insert a single row into the ``cities`` table::
 Insert multiple rows into the ``cities`` table::
 
     INSERT INTO cities VALUES (2, 'San Jose'), (3, 'Oakland');
+
+Insert a sinle row into the ``nation`` table with usage of the column list::
```

Comment:
Let's remove all the `_n` prefixes from the example columns, since it makes the examples harder to read.